### PR TITLE
Edit Help Window

### LINKED ISSUE #117: Update Help Window with latest features

### PR DESCRIPTION
**Pull request by pennhanlee**
closes #117 
 
* Updated Helpwindow to have a oldpaper textured background
* Added the bookmark icon to the headerbar
* Updated the list of commands and the output 
    * Commands Titles
    * Command Message (So what)
    * Command Example

Attached is a screenshot of the current updated help window. 
<p align="center">

</p>

* Typo in List Command message has been editted

### CODE REVIEW

File: src/main/java/seedu/bookmark/ui/help/HelpAdd.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
+package seedu.bookmark.ui.help;
+
+public class HelpAdd implements HelpAction {
+
+    private static final String ADD_INTRO = "ADD A BOOK!";
+    private static final String ADD_MESSAGE = "Add books to the storage to keep track of your "
+                                                + "reading progress using our Add Command!";
+    private static final String ADD_COMMAND = "COMMAND: add n/BOOK_TITLE g/GENRE [t/TAG] [t/TAG] tp/TOTAL_PAGES "
```
Comment (TanLeYang) [inline src/main/java/seedu/bookmark/ui/help/HelpAdd.java]: Is there supposed to be two [t/TAG] here to show that users can add multiple tags? If would it be better to explain it in the ADD_NOTES instead?

**Review by TanLeYang — COMMENTED** [review]
Other than the one small issue noted, LGTM!

File: src/main/java/seedu/bookmark/ui/help/HelpAdd.java (hunk 1 above)
Comment (pennhanlee) [reply src/main/java/seedu/bookmark/ui/help/HelpAdd.java]: Good Idea! i've made the changes. Thank you :)

**Review by TanLeYang — APPROVED** [review]
LGTM!

**Pull request merged**